docs/token-contracts/fa2/2-fa2-nft-tutorial.md

```diff
@@ -0,0 +1,533 @@
+---
+id: 2-fa2-nft-tutorial
+title: Non-Fungible Tokens on Tezos Using FA2
+sidebar_label: FA2 NFT Tutorial LIGO
+---
+
+This tutorial shows how to originate and interact with the FA2 NFT contract
+implementation. The tutorial uses a pre-compiled FA2 NFT contract written in
+[LIGO](https://ligolang.org/) smart contract language and a command line interface
+(CLI) to originate and interact with the NFT contracts either on the
+[Flextesa](https://tezos.gitlab.io/flextesa/) sandbox or Tezos testnet (Carthagenet).
+
+**Disclaimer:** We highly recommend users read the additional resources above and
+take necessary precautions before following this tutorial and interacting with
+experimental technology. Use this tutorial at your own risk.
```

Comment:
Is such a strong disclaimer necessary? `tnft` and the other included software include disclaimers in their licenses, e.g. https://github.com/tqtezos/nft-tutorial/blob/master/LICENSE

Maybe something along the lines of "we assume basic familiarity with LIGO and Flextesa: see those links for more detail"?